raiden-dapp/src/locales/en.json

```
@@ -365,6 +362,12 @@
       "title": "Token transfer unsuccessful",
       "deposit-title": "Deposit failed, please try again."
     },
+    "transfer-menus": {
+      "change-token-title": "Change Token",
+      "deposit-button": "Deposit",
+      "channel-button": "Channels",
+      "no-channels": "No Channels"
```

```suggestion
      "no-channels": "No open channels"
```

There might be still a closing and closed channels. Maybe also adopt the „variable“ naming for consistency.

---

Yes, good point.